spec/support/lesli_request_tester.rb

```diff
@@ -60,8 +60,8 @@
             }
         }
 
-        @current_user = User.first
-        sign_in(@current_user)
+        @current_user = Lesli::User.first
+        #sign_in(@current_user)
```

Comment:
is there a new way to "sign in" the user we'll use for each test case ? 🤔

Reply:
We need LesliVault ready to be able to work with authentication :)

Reply:
jajajaj